Makefile.am

```diff
@@ -1,9 +1,12 @@
+SUBDIRS = \
+	package_modules \
+	tests/unit
```

Comment:
Actually, ``TESTS_SUBDIRS`` should not be there at all, should just assign directly to ``SUBDIRS``. Will fix.